manifests/service.pp

```diff
@@ -7,11 +7,37 @@
 #
 # * Justin Lambert <mailto:jlambert@letsevenup.com>
 #
-class kibana::service {
+class kibana::service (
+  $service_provider    = $::kibana::service_provider,
+) {
 
-  service { 'kibana':
-    ensure => running,
-    enable => true,
+  if $service_provider == 'init' {
```

Comment:
I agree the service duplication isn't good but I still think that the init script belongs in ```service.pp``` since it relates more to the 'service' than it does to 'install'.

Also I think that this part of the module might get more complex if we start to support other OSs (we will need an Debian style SysV init script too, for example) and ```service.pp``` seems like a better home for this logic.

What do you think of the fix in my latest commit which removes the duplication but keeps the init file inside ```service.pp```?